src/renderer/Components/CreatingPackage/CreatingOrder/BookmarkToolbar/BookmarkList/BookmarkList.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from "prop-types";
+
+import usePackageStore from "@renderer/store";
+
+function BookmarkList({ bookmarks, closeBookmarkList }) {
+  const { updateOrder } = usePackageStore();
+
+  const applyBookmark = (index) => {
+    delete bookmarks[index]._id;
+    delete bookmarks[index].updatedAt;
+    delete bookmarks[index].createdAt;
+
+    updateOrder(bookmarks[index]);
+    closeBookmarkList();
+  };
+
+  return (
+    <>
+      {bookmarks.map((bookmark, index) => (
+        <button
+          key={bookmark.createdAt}
```

Comment:
해당 key 값을 createdAt 으로 주고 계시데 혹시 겹치가능성은 없나여?🤔
저는 데이터베이스에서 bookmarks 가져오고 있으니 _id 속성으로 주시면 확실하게 안 겹칠것 같습니다.😄
```suggestion
          key={bookmark._id}
```

Reply:
겹칠 가능성은 적다고 생각됩니다만, 혹시 몰르 경우를 대비해 _id 값을 사용하는 게 좋을 것 같습니다. 반영하겠습니다. 🤭